canonical/canonical.com
# Microcloud roadshow migration

## Done

- Migrate https://microcloud.is/roadshow to c.com 

## QA

- View the site locally in your web browser at: http://0.0.0.0:8002/
- Compare against live page and see that everything looks as expected (modal won't work until revamped version is merged into feature branch)

## Issue / Card
